pmd-plsql/src/test/java/net/sourceforge/pmd/lang/plsql/ast/PLSQLParserTest.java

```diff
@@ -73,4 +73,9 @@ public void testCodingStyleExample() {
     public void testCaseIssue1454() {
         plsql.parseResource("CaseIssue1454.pls");
     }
+
+    @Test
+    public void testCaseIssue3106() {
```

Comment:
Test name should be something like "testExecuteImmediateIssue3106" - the test before (#1454) is indeed about the case statement.... but #3106 is about execute immediate.

Please put the test source in a separate file, so that we can better read it. In that file, you can use multiple lines and don't need escapes.